deploy/olm-catalog/ibm-block-csi-operator-community/1.11.0/manifests/ibm-block-csi-operator.v1.10.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,657 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
could you change the file name to `ibm-block-csi-operator.v1.11.0.clusterserviceversion.yaml` insted of `ibm-block-csi-operator.v1.10.0.clusterserviceversion.yaml`?

Reply:
Done